### PR TITLE
add a test for ios native to capture nav bar

### PR DESCRIPTION
**Pull request by SophieDePaula**
This test was created due to [this PR](https://github.com/applitools/eyes.sdk.javascript1/pull/848)

### CODE REVIEW

File: java/overrides.js [hunk 1]
```diff
@@ -53,4 +53,6 @@ module.exports = {
       
     // TODO verify and enable
     'should send agentRunId': {skipEmit: true},
+    'appium iOS nav bar check regio': {skipEmit: true},
```
Comment (Yarden-Ingber) [inline java/overrides.js:56]: Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/applitools/sdk.coverage.tests/pullRequest?prid=9214167)
```suggestion
    "appium iOS nav bar check regio": {skipEmit: true},
```

File: python/overrides.js [hunk 2]
```diff
@@ -22,4 +22,5 @@ module.exports = {
     // TODO verify and enable
     'appium iOS check fully window with scroll and pageCoverage': { skipEmit: true },
     'appium iOS check window region with scroll and pageCoverage': { skipEmit: true },
+    'appium iOS nav bar check regio': {skipEmit: true},
```
Comment (Yarden-Ingber) [inline python/overrides.js:25]: Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/applitools/sdk.coverage.tests/pullRequest?prid=9214167)
```suggestion
    "appium iOS nav bar check regio": {skipEmit: true},
```

File: ruby/overrides.js [hunk 3]
```diff
@@ -45,4 +45,5 @@
     'should waitBeforeCapture with breakpoints in check': { skipEmit: true },
     'should waitBeforeCapture with breakpoints in open': { skipEmit: true },
     'should be empty if page delayed by 1500': { skipEmit: true },
+    'appium iOS nav bar check regio': {skipEmit: true},
```
Comment (Yarden-Ingber) [inline ruby/overrides.js:48]: Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/applitools/sdk.coverage.tests/pullRequest?prid=9214167)
```suggestion
    "appium iOS nav bar check regio": {skipEmit: true},
```

File: coverage-tests.js [hunk 4]
```diff
@@ -2235,4 +2235,15 @@ test('should send agentRunId', {
     assert.equal(info1.startInfo.agentRunId, info2.startInfo.agentRunId)
   },
 })
+
+test('appium iOS nav bar check region', {
+  env: {device: 'iPhone XS', app: 'https://applitools.jfrog.io/artifactory/Examples/awesomeswift.zip'},
+  config: {baselineName: 'Appium_iOS_CheckRegion'},
```
Comment (tourdedave) [inline coverage-tests.js]: `baselineName` shouldn't be needed. It's constructed from the test name dynamically.

File: coverage-tests.js [hunk 5]
```diff
@@ -2235,4 +2235,15 @@ test('should send agentRunId', {
...
+  features: ['native-selectors'],
+  test: ({driver, eyes, helpers, assert}) => {
+    eyes.open({appName: 'Applitools Eyes SDK', testName: 'Appium_iOS_Nav_CheckRegion'})
```
Comment (tourdedave) [inline coverage-tests.js]: We shouldn't need to pass the `testName` explicitly here

**Review by tourdedave — CHANGES_REQUESTED** [review]
Looks good other than the tweaks I mentioned.

If you haven't already, you'll also want to run the test and accept the new baseline in Eyes on the `Eyes SDK` team.

**Comment by SophieDePaula** [comment]
Thanks @tourdedave

**Pull request merged**